servicetest/dbus/dbusapp.py

```diff
@@ -60,7 +60,7 @@ def Bounce(self, message):
 
     @dbus.service.method(IFACE, in_signature="s", out_signature="s")
     def Ping(self, message):
-        print "Got a Ping: ", message
+        print(("Got a Ping: ", message))
```

Comment:
I prefer `print("Got a Ping {}".format(message))` as it explicitly shows where the message will be inserted